### ISSUE TITLE
Telefon Online Offline Status senden?

### ISSUE THREAD

**Issue by merlink01** [sipgate/sipgate.io#105]
Ist es möglich den Online/offline Status eines Telefons per puch nachricht zu senden?

Somit könnte unsere Liste, welcher Mitarbeiter frei zum sprechen ist erweitert werden.
Grau = offline
Grün = Frei
Gelb = Klingelt
Rot = Besetzt

Fals das Technisch möglich ist ;-)

**Comment by CookieCrumb23** [comment]
Das ist technisch leider etwas aufwändiger und momentan nicht in unserem Fokus, deswegen kann ich auch nicht sagen ob und wann das kommen könnte.

**Comment by danielberlin** [comment]
Auf Snom-Geräten kann man unter `<ip>/action.htm` u.a. folgendes definieren:

http://wiki.snom.com/wiki/index.php/Settings/action_log_on_url
http://wiki.snom.com/wiki/index.php/Settings/action_log_off_url
http://wiki.snom.com/wiki/index.php/Settings/action_dnd_on_url

Hier lässt sich also ein eigenes Skript eintragen und damit die Liste, wer frei ist, kinderleicht erstellen.

Interessant vor allem auch als Monitoring – reg_failed sollte nie passieren:
http://wiki.snom.com/wiki/index.php/Settings/action_reg_failed

**Comment by merlink01** [comment]
Ja, das ist klar. Nur wäre eine einheitliche Lösung natürlich flexibler. 
Was macht man z.B. bei sim Karten oder wenn man voip - isdn umsetzer verwendet.
Dies wäre ein cooles feature das sipgate definitiv von anderen Herstellern abheben würde.
